Description: Debian specific tweaks for NetworkManager systemd service file
 This patch is needed to avoid a dependency loop which is generated
 by early boot (rcS) SysV init scripts depending on the $network LSB
 system facility (either directly or indirectly via $remote_fs).
 .
 The $network LSB system facility is mapped to network.target.
 .
 Those rcS SysV init scripts are started in sysinit.target,
 NetworkManager.service is started via multi-user.target, which depends
 on sysinit.target.
 .
 As long as we have rcS SysV init scripts which such a dependency on
 $remote_fs or $network we can not make NetworkManager.service a provider
 for network.target.

Author: Michael Biebl <biebl@debian.org>
--- a/data/NetworkManager.service.in
+++ b/data/NetworkManager.service.in
@@ -1,7 +1,5 @@
 [Unit]
 Description=Network Manager
-Wants=network.target
-Before=network.target
 
 [Service]
 Type=dbus
